python/ray/data/dataset_pipeline.py

```diff
@@ -642,6 +647,11 @@ def __iter__(self):
 
         return EpochDelimitedIterator(self)
 
+    def _peek(self) -> (Dataset[T], Iterator[Dataset[T]]):
```

Comment:
Move this to the end of the class definition?

Reply:
Done.